src/renderer/pages/Contact/ContactAll.vue

```diff
@@ -176,18 +176,18 @@ export default {
       return sortBy(contacts, ['name', 'address'])
     },
 
-    hasGridLayout () {
-      return this.$store.getters['session/hasGridLayout']
+    hasWalletGridLayout () {
+      return this.$store.getters['session/hasWalletGridLayout']
     },
 
-    sessionLayout: {
+    walletLayout: {
       get () {
-        return this.$store.getters['session/layout']
+        return this.$store.getters['session/walletLayout']
       },
       set (layout) {
-        this.$store.dispatch('session/setLayout', layout)
+        this.$store.dispatch('session/setWalletLayout', layout)
```

Comment:
Why not adding an action to the profile store and use it in both contacts and wallets?

Reply:
This is used in both the wallets and contacts. Contacts are still technically wallets, just not yours, so I thought this was a better name than just "layout". We could split it into `setWalletLayout` and `setContactLayout` in the future if people have different requirements for each I think

Reply:
Thanks, @alexbarnsley, fo the explaination.